src/coreComponents/mesh/EmbeddedSurfaceSubRegion.cpp

```diff
@@ -63,10 +64,25 @@ EmbeddedSurfaceSubRegion::EmbeddedSurfaceSubRegion( string const & name,
     setRTTypeName( rtTypes::CustomTypes::groupNameRefArray ).
     setDescription( "A map of surface element to the parent fracture name" );
 
+  registerWrapper( viewKeyStruct::surfaceElementsToCellRegionsString(), &m_2dElemToElems.m_toElementRegion ).
+    setPlotLevel( PlotLevel::NOPLOT ).
+    setDescription( "A map of face element local indices to the cell local indices" );
+
+  registerWrapper( viewKeyStruct::surfaceElementsToCellSubRegionsString(), &m_2dElemToElems.m_toElementSubRegion ).
+    setPlotLevel( PlotLevel::NOPLOT ).
+    setDescription( "A map of face element local indices to the cell local indices" );
+
+  registerWrapper( viewKeyStruct::surfaceElementsToCellIndexString(), &m_2dElemToElems.m_toElementIndex ).
+    setPlotLevel( PlotLevel::NOPLOT ).
+    setDescription( "A map of face element local indices to the cell local indices" );
+
   m_normalVector.resizeDimension< 1 >( 3 );
   m_tangentVector1.resizeDimension< 1 >( 3 );
   m_tangentVector2.resizeDimension< 1 >( 3 );
   m_2dElemToElems.resize( 0, 1 );
+
+  m_2dElemToElems.setElementRegionManager( dynamicCast< ElementRegionManager & >( getParent().getParent().getParent().getParent() ) );
```

Comment:
This is one of those situations I'm unsure whether we want absolute or relative addresses or e.g. a member function on a "static" Group/Node to setup relationships between Group/Nodes in the DataRepo/Graph, I feel like we're inconsistent with this and we should probably have some guidance/best practices related to this, not important for this PR but might be something to discuss at an infra meeting @rrsettgast

Reply:
We can just move this somewhere that has access to `MeshLevel`...that would be better.